src/go/k8s/pkg/utils/podutils.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 Redpanda Data, Inc.
```

Comment:
NIT:
```suggestion
// Copyright 2022-2023 Redpanda Data, Inc.
```

Reply:
> The general rule is that the year to include in a copyright notice is the year of first publication of the work.

https://www.copyrightlaws.com/copyright-symbol-notice-year/#h-which-year-should-you-use-in-your-copyright-notice